CONTRIBUTORS.md

```diff
@@ -1,5 +1,6 @@
 Cerner Corporation
 - Bharath Banothu [@bbanothu]
+- Bharath Banothu [@bbanothu]
```

Comment:
This looks like a duplicate entry.

Reply:
Fixed, I thought I needed to reAdd my name again for a new merge.